core/backend/package.json

```diff
@@ -101,6 +101,8 @@
   "dependencies": {
     "@bentley/imodeljs-native": "5.0.41",
     "@itwin/cloud-agnostic-core": "^2.2.4",
+    "@itwin/ecschema-locaters": "workspace:*",
+    "@itwin/ecschema-metadata": "workspace:*",
```

Comment:
should this be a peer instead of direct dep?

Reply:
True, peer dependency seems like the better choice. I'll change it

Reply:
done